configuration/rules-dsl.md

```diff
@@ -21,10 +21,14 @@ On this page you will learn how to leverage its functionality to do *real* home
 
 ### File Location
 
-Rules are placed in the folder `${openhab.home}/conf/rules`. The [demo setup]({{base}}/tutorials/demo.html) already comes with a demo file called `demo.rules`, which has a couple of examples that can be a good starting point.
+Rules are placed in the folder `${openhab.home}/rules`. The [demo setup]({{base}}/tutorials/demo.html) already comes with a demo file called `demo.rules`, which has a couple of examples that can be a good starting point.
```

Comment:
Good one. Even better would be if you could follow #189 and use `$OPENHAB_CONF/rules`.

Reply:
Yes, this needs to be changed.  But I don't remember how I made the edits on this PR as I was not set up yet with GitHub Desktop at that time.  Can you go ahead and make this change?